src/ui/sass/modules/_ResultsHeader.scss

```diff
@@ -82,7 +84,7 @@
   &-filterSeparator
   {
     color: $results-header-filters-color;
-    margin-right: $results-header-spacing * 3 / 8;
+    margin-right: $results-header-horizontal-spacing;
```

Comment:
Isn't results-header-horizontal-spacing 0.5 * results-header-spacing? The previous styling was 3/8 * results-header-spacing. Was that incorrect?

Reply:
looking at the older mocks it looks like this was incorrect. I want to say I saw in the mocks that the spacing was 6px instead of the 8px it currently is (I'm sure I didn't make that weird fraction up on my own) but I can't find a mock showing this